saleor/core/search_tasks.py

```diff
@@ -15,7 +15,7 @@
 
 task_logger = get_task_logger(__name__)
 
-BATCH_SIZE = 1000
```

Comment:
Why?

Reply:
Reduced it by half because we found out it was timing out with the larger batch.

Reply:
It's a magic number that someone will change next week. If it needs to be 500, you have to add a long comment explaining why it needs to be 500 and how to retest it in a year to ensure it's still the correct threshold. Otherwise, it'll either get randomly changed back to 1000 or stuck at 500 for ten years from the fear of breaking something.